benchmarks/performance_test.py

```diff
@@ -1,17 +1,78 @@
 from urlparse4 import urlsplit, urljoin
 from timeit import default_timer as timer
 
-total = 0
+from sys import argv
 
-with open('urls/chromiumUrls.txt') as f:
-    for url in f:
+encode = False
 
-        start = timer()
+try:
+    if argv[1] == "encode":
+        encode = True
+except IndexError:
+    print("encode is not defined, continue with the program...")
 
-        a = urlsplit(url)
+if encode:
+    total = 0
 
-        end = timer()
+    for i in range(5):
+        with open('urls/chromiumUrls.txt') as f:
+            for url in f:
 
-        total += end - start
+                start = timer()
 
-print("the total time is", total, "seconds")
+                a = urlsplit(url.encode())
+
+                end = timer()
+
+                total += end - start
+
+    print("the urlsplit time with encode in python is", total / 5, "seconds")
+
+
+    total2 = 0
```

Comment:
maybe ``urljoin_time`` and ``urlsplit_time`` would be better than ``total`` and ``total2``